gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/TraversalStrategies.java

```diff
@@ -215,7 +216,8 @@ static void visit(Map<Class<? extends TraversalStrategy>, Set<Class<? extends Tr
                     PathRetractionStrategy.instance(),
                     LazyBarrierStrategy.instance(),
                     ProfileStrategy.instance(),
-                    StandardVerificationStrategy.instance());
+                    StandardVerificationStrategy.instance(),
+                    ByModulatorOptimizationStrategy.instance());
```

Comment:
Best to put this in the right location. Thus, not AFTER the verification strategies, but in the optimization strategies.